EasyPost.Tests/InsuranceTest.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyPost.Tests
+{
+    [TestClass]
+    public class InsuranceTest
+    {
+        private Dictionary<string, object> _fromAddressData { get; set; }
+        private Insurance _insurance { get; set; }
+        private Dictionary<string, object> _parcelData { get; set; }
+        private Dictionary<string, object> _toAddressData { get; set; }
+        private Dictionary<string, object> InsuranceData { get; set; }
```

Comment:
**InsuranceData** is a private variable, should the variable start with an underscore and lowercase?

Reply:
Good point, resolved in [76da6a1](https://github.com/EasyPost/easypost-csharp/pull/205/commits/76da6a1c7800ac4df98f8c230458ee78540999bc)

Reply:
Why are the private variables all start with uppercase now?

I am following the naming convention from the Microsoft doc: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

Reply:
These are properties (setters and getters), not fields.

Reply:
thanks for the clarification.